test/core/event_engine/test_suite/BUILD

```diff
@@ -57,24 +80,30 @@ grpc_cc_library(
     name = "server",
     testonly = True,
     srcs = ["server_test.cc"],
-    hdrs = COMMON_HEADERS,
+    external_deps = [
+        "absl/status",
+        "absl/status:statusor",
+        "absl/strings",
+        "absl/time",
+        "gtest",
+    ],
     deps = [
-        ":conformance_test_base_lib",
+        "conformance_test_base_lib",
+        "//:exec_ctx",
+        "//:grpc",
         "//src/core:channel_args",
+        "//src/core:channel_args_endpoint_config",
+        "//src/core:memory_quota",
+        "//src/core:notification",
+        "//src/core:resource_quota",
+        "//test/core/util:grpc_test_util_unsecure",
     ],
     alwayslink = 1,
 )
 
 grpc_cc_library(
     name = "complete",
     testonly = 1,
-    hdrs = COMMON_HEADERS,
-    deps = [
-        ":client",
```

Comment:
`fix_build_deps` is removing all of the actual tests from all of the test targets. Can the tool be disabled here?

Reply:
made the actual tests `nofixdeps`, removed `complete`